src/test/java/com/example/employeeTDD/demo/BDD_Cucumber_Tests/GetEmployee.java

```diff
@@ -0,0 +1,58 @@
+package com.example.employeeTDD.demo.BDD_Cucumber_Tests;
+
+import com.example.employeeTDD.demo.Controller.EmployeeController;
+import com.example.employeeTDD.demo.Entity.Employee;
+import com.example.employeeTDD.demo.Repository.EmployeeRepository;
+import com.example.employeeTDD.demo.Service.EmployeeService;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.function.Executable;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+public class GetEmployee {
+    @MockBean
+    private EmployeeService service;
+    @MockBean
+    private EmployeeRepository repository;
+    @MockBean
+    private EmployeeController controller;
+
+    public GetEmployee() {
+
+
+        Employee employee =new Employee(1, "Ed","Hardy","eh@gmail.com" );
+
+        Given(("^there is a get employee request based on ID$", () -> {
```

Comment:
Write separate methods for the scenarios rather than put everything in the constructor

Reply:
acknowledged - 8aea984